README.md

```diff
@@ -139,6 +140,32 @@ Inside the link entity in your model, add a custom sensor:
       </vertical>
   </pattern_uniform>
   ```
+  **Note:** - Optionally for 2D lidar patterns (vertical samples = 1), you can use the `<publish_laserscan>` param like this
+  ```xml
+  <sensor name="UniformPattern2DLidar" type="custom">
+    <plugin filename="RGLServerPluginInstance" name="rgl::RGLServerPluginInstance">
+      <range>100</range>
+      <update_rate>10</update_rate>
+      <update_on_paused_sim>false</update_on_paused_sim>
+      <topic>rgl_lidar</topic>
+      <frame>RGLLidar</frame>
+      <pattern_uniform>
+        <horizontal>
+            <samples>1800</samples>
+            <min_angle>-3.14159</min_angle>
+            <max_angle>3.14159</max_angle>
+        </horizontal>
+        <vertical>
+            <samples>1</samples>
+            <min_angle>-0.436332</min_angle>
+            <max_angle>0.261799</max_angle>
+        </vertical>
+    </pattern_uniform>
+    <publish_laserscan>true</publish_laserscan>
```

Comment:
Definitely a better idea. I went the `publish_laserscan` route because I know of a few use cases where It would be beneficial to have a point cloud even if it is 2d. Will try this method.

Reply:
> (...) use cases where It would be beneficial to have a point cloud even if it is 2d

There will be still a way to define 2d lidar using patterns for 3d lidars and get point cloud. I wonder if it would be useful to publish both types of messages, but I've never encountered such a case.